### ISSUE TITLE
Restoring a window sets it to the default size

### ISSUE THREAD

**Issue by WPFerg** [ScottLogic/StockFlux#619]
Resize a window, maximise, restore. It'll be back to the default height, not the height it was pre-maximise.

**Comment by WPFerg** [comment]
This seems to be part of a larger issue, in that without that code to do that, a window created in compact and then expanded is not resizable.
Having a further look at the information passed to openfin, via: `_window.updateOptions(newOptions, cb)`, where `cb` is:
```js
() => {
    _window.getOptions((opts) => {
        console.log('got opts', opts);
    });
};
```
Openfin outputs this state for when the window is *not* compact, ie. that it *should* be resizable: `
resizable: true`. However, it is not resizable.

**Comment by WPFerg** [comment]
Just thought I should add some additional context here.
https://github.com/WPFerg/OpenFinD3FC/commit/5e55140b617387a7f850b916efa99e016ecfe996#diff-4664b655695bc5923ad584bb3d9b3107L62

This is the line in question, the issue that this fixes can be reproduced as follows (assuming when uncompacting from a maximised window maximises the window -- the fix in PR #718 -- is implemented):
* Maximise a window
* Compact
* Uncompact
* Restore
